test/support/publish_dsl.rb

```diff
@@ -25,11 +25,191 @@ def publish_wait
       page.wont_have_css 'publish-spinner'
     end
 
+    def publish_wait_for_title
+      find('#title')
+    end
+
+    def publish_wait_for_episodes_link
+      find('a', text: '0 Episodes in Series')
+    end
+
+    def publish_wait_for_success
+      find('div.success')
```

Comment:
For some reason, I'm always getting `Capybara::ElementNotFound` on this running locally.  But it works fine in docker.  Maybe my phantom version is off or something?

Reply:
I have phantomjs 2.1.1 locally fwiw. I haven't tried docker.